README.md

```diff
@@ -28,65 +28,72 @@ Alternatively, they can be integrated `annotate-integrate-annotations` as commen
 
 - annotations in org-mode source blocks will be underlined, but the annotations don't show up. This is likely a fundamental incompatibility with the way source blocks are highlighted and the way annotations are displayed.
 
+Portion of the code © 2019 Universita' degli Studi di Palermo
+
 This package is released under the MIT license.
 
+
+
 ### Changelog
 
-- **2015-06-12 V0.1 Bastian Bechtold**  
+- **2015-06-12 V0.1 Bastian Bechtold**
```

Comment:
Please do not delete these white spaces. They are meaningful in markdown (also applies to all following lines in this file).